src/Bundle/CoverallsBundle/Console/ApplicationSf6.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PhpCoveralls\Bundle\CoverallsBundle\Console;
+
+use PhpCoveralls\Bundle\CoverallsBundle\Command\CoverallsJobsCommand;
+use Symfony\Component\Console\Application;
+use Symfony\Component\Console\Input\InputDefinition;
+use Symfony\Component\Console\Input\InputInterface;
+
+/**
+ * Coveralls API application for Symfony 6 and higher
+ *
+ * @author Viktor Linkin <adrenalinkin@gmail.com>
+ */
+class ApplicationSf6 extends Application
```

Comment:
I wonder, can we extend `PhpCoveralls\Bundle\CoverallsBundle\Console\Application` here? Maybe it would let us to minify amount of code. (or maybe we cannot due to typing conflicts?)

Reply:
I will investigate this opportunity